context/llpcGraphicsContext.cpp

```diff
@@ -186,6 +182,29 @@ const PipelineShaderInfo* GraphicsContext::GetPipelineShaderInfo(
     return pShaderInfo;
 }
 
+// =====================================================================================================================
+// Get the final vertex processing shader stage in this pipeline, or ShaderStageInvalid if none.
+ShaderStage GraphicsContext::GetFinalVertexStage() const
```

Comment:
GetFinalVertexStage -> GetLastVertexProcessingStage
We have encounter this naming problem before and use this terminology from OpenGL spec to express this idea.

Reply:
OK, I'll change to the OpenGL spec convention. More generally, "final" is better than "last" because "last" can also mean "previous". :-)